src/Microsoft.AspNet.Mvc.Core/ActionConstraints/ConsumesAttribute.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNet.Mvc.HeaderValueAbstractions;
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// Specifies the allowed content types which can be used to help in selecting the action based on request's
```

Comment:
`which can be used to help in selecting the action` -> `which can be used to select the action`